.github/workflows/publish.yml

```
@@ -0,0 +1,125 @@
+name: Publish Package
+on: 
+  workflow_dispatch:
+    inputs:
+      revision:
+        description: 'Revision'
+        required: true
+        default: 'prerelease'
+        type: choice
+        options:
+        - prerelease
+        - minor
+        - major
+        - patch
+jobs:
+  publish-mangrove-js:
```

mangrove.js seems misplaced here :-)

Maybe the job could be called something like
```suggestion
  publish-npm-package:
```

---

True ahah